aea/helpers/ipfs/base.py

```diff
@@ -92,7 +92,7 @@ def get(cls, file_path: str, wrap: bool = True, cid_v1: bool = True) -> str:
         if os.path.isdir(file_path):
             return cls.hash_directory(file_path, wrap=wrap, cid_v1=cid_v1)
 
-        return cls.hash_file(file_path, wrap=wrap)
```

Comment:
The answer would be no since it already defaulted to `True` 
https://github.com/valory-xyz/open-aea/blob/bfe783f040a044ea4bf46a7ea3728ede97ae7df8/aea/helpers/ipfs/base.py#L95-L98